x-pack/test/performance/journeys/base.config.ts

```diff
@@ -60,7 +60,7 @@ export default async function ({ readConfigFile, log }: FtrConfigProviderContext
       ...functionalConfig.get('kbnTestServer'),
       serverArgs: [
         ...functionalConfig.get('kbnTestServer.serverArgs'),
-        `--telemetry.optIn=true`,
+        `--telemetry.optIn=${process.env.TEST_PERFORMANCE_PHASE === 'TEST'}`,
```

Comment:
Can we also pass `'--telemetry.banner=false',` to make sure that the opt-in banner isn't rendered in the WARMUP phase?

Reply:
Ohh this branch was not synced with upstream, I will merge upstream but did you mean this:
https://github.com/elastic/kibana/pull/140074/files#diff-35bdbcece3f4681b2f59ed2a105845187e2729fb2d0778208ec064acc4514863R65-R66
Or '--telemetry.banner=false' which I'm not familiar what that is

Reply:
I mean `--telementry.banner`, when you haven't opted-into telemetry a banner is shown at the top of the app which causes content to be shifted down, which can cause certain buttons to be off screen in the warmup phase of the tests.

@elastic/kibana-telemetry do we need to pass `--telemetry.banner=false` in order to hide this banner when we pass `--telemetry.optIn=false`?

Reply:
It's being explicity disabled [here](https://github.com/suchcodemuchwow/kibana/blob/disable-telemetry-for-warmup-phase/test/common/config.js#L47) but would be nice to hear from @elastic/kibana-telemetry too

Reply:
Oh, yeah, didn't realize we were extending that config.

Reply:
Shall we wait for response or default config is good to go ?

Reply:
I think the fact that we're extending the default config is sufficient.